readme/usage.md

```diff
@@ -85,7 +85,17 @@ Examples:
 - Alphanumeric `a` to `z` and `0` to `9`
 - Modifiers `Alt_L` `Control_L` `Control_R` `Shift_L` `Shift_R`
 - Mouse buttons `BTN_LEFT` `BTN_RIGHT` `BTN_MIDDLE` `BTN_SIDE` ...
-- Multimedia keys `KEY_NEXTSONG` `KEY_PLAYPAUSE` ...
+- Mouse scroll `wheel(down, 1)` `wheel(up, 1)`
+- Mouse move `mouse(left, 1)` `mouse(right, 1)` `mouse(up, 1)` `mouse(down, 1)`
+- Multimedia keys `KEY_NEXTSONG` `KEY_PLAYPAUSE` `XF86AudioMicMute` ...
+
+## Keyboard/Mouse cooperation
+
+Left click mapped to a keyboard key can be held while moving the cursor by
+a real mouse/trackball, resulting in proper drag'n'drop.
```

Comment:
This isn't something that key-mapper implemented, it's just how the environment uses the provided events mapped by the user. If a BTN_LEFT can be mapped then it would be expected that it behaves just like a regular mouse button

also this is kinda an example, it could be also said about other combinations of injected events like *shift on device 1* + *a on device 2*. What is it about the *btn_left on device 1* + *movement on device 2* combination that is so special that it should be mentioned?